torch/onnx/symbolic_opset9.py

```diff
@@ -1252,7 +1252,8 @@ def exp(g, self):
 
 @parse_args('v', 'f', 'i')
 def dropout(g, input, p, train):
-    if not train:  # in eval mode, dropout is non-op
+    # in eval mode, dropout is non-op - if the node's train param is set to False, dropout is non-op
+    if not sym_help._training_mode:
         return input
     warnings.warn("Dropout is a training op and should not be exported in inference mode. "
```

Comment:
Can we update this warning to be more informative about how we cannot export Dropout in training mode for opsets < 12?